### PR TITLE
Feature/saving and loading objects

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/Storage.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+public class Storage {
```
Comment (vadymmarkov) [inline Source/Storage.swift]: Could we use `struct` here?

File: Source/Storage.swift [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  static var fileManager = {
+    return NSFileManager.defaultManager()
+    }()
+
+  static let applicationDirectory: String = {
+    let paths:NSArray = NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true)
+    let basePath: AnyObject! = (paths.count > 0) ? paths.firstObject : nil
```
Comment (vadymmarkov) [inline Source/Storage.swift]: It's kind of weird that `AnyObject!` could be `nil` at some moment.

File: Source/Storage.swift [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    return buildPath
+  }
+
+  static func load(path: URLStringConvertible) -> AnyObject? {
+    let loadPath = Storage.buildPath(path, createPath: false)
+    return fileManager.fileExistsAtPath(loadPath)
+      ? NSKeyedUnarchiver.unarchiveObjectWithFile(loadPath)
+      : nil
```
Comment (vadymmarkov) [inline Source/Storage.swift:38]: Looks nice! :sunglasses:

File: Source/Storage.swift (hunk 1 above)
Comment (zenangst) [inline Source/Storage.swift]: Fixed https://github.com/zenangst/Storage/commit/791f27283c0f0a8918eb43ec9b1848b5e4ada3f5

File: Source/Storage.swift (hunk 2 above)
Comment (zenangst) [inline Source/Storage.swift]: Fixed https://github.com/zenangst/Storage/commit/e9ee3a9845af799897597ebe993f4b4986e183a8

**Comment by zenangst** [comment]
@vadymmarkov better? :sunglasses:

**Comment by vadymmarkov** [comment]
@zenangst :cake:

**Pull request merged**